coremltools/converters/mil/frontend/tensorflow/test/test_composite_ops.py

```diff
@@ -30,10 +30,10 @@ def create_custom_selu(self):
         @register_tf_op(tf_alias=[], override=True)
         def Selu(context, node):
             x = context[node.inputs[0]]
-            alpha = 1.6732631921768188
-            lamda = 1.0507010221481323
-            out_elu = mb.elu(x=x, alpha=alpha)
-            out = mb.mul(x=out_elu, y=lamda, name=node.name)
+            α = 1.6732631921768188
```

Comment:
I don't think using non-ascii character is a common practice. Could you revert the change here?